app/templates/components/paper-card-actions.hbs

```diff
@@ -0,0 +1,3 @@
+<md-card-actions class="{{class}} layout-{{layout}} layout-align-{{layoutAlignClassName}}">
```

Comment:
I don't think we need to provide specific `layout ` and `layoutAlign` properties.
Won't these flex classes be part of the public API?